frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebookSquare } from '@fortawesome/free-brands-svg-icons';
+import '../../src/styles/base/typography.css';
+import '../../src/styles/base/variables.css';
+import '../../src/styles/components/footer.css';
```

Comment:
Perhaps we can remove css imports from JSX files and just import them to the styles.css

Reply:
This has been addressed.